src/monitor/other.rs

```diff
@@ -0,0 +1,7 @@
+use super::*;
+
+pub fn create_signal_monitor() -> impl Future<Item = (), Error = io::Error> + Send {
+    // FIXME: What can I do ...
+    // Blocks forever
+    futures::empty::<(), ()>().and_then(|_| {})
```

Comment:
I'm not sure the `.and_then(|_| {})` part is needed? Looks like you can make a future that fits the expected signature directly from `futures::empty::<(), io::Error>()`?

Reply:
Would that give the same behavior? The future isn't supposed to ever complete.

Reply:
Yes. The docs for `futures::empty` says "A future which is never resolved.". So `futures::empty::<T, E>()` would return a future that never completes and that implements `Future<Item=T, Error=E>`.